configure.ac

```diff
@@ -238,6 +238,12 @@
             #include <sys/random.h> 
             ])
 
+    AC_CHECK_DECL([memmem],
```

Comment:
why not use `AC_CHECK_FUNCS([memmem])`

Reply:
I guess because I am no auto tools expert :-p
Changing it